KubeArmor/monitor/systemMonitor.go

```diff
@@ -215,6 +249,150 @@ func isIgnored(item string, ignoreList []string) bool {
 	return false
 }
 
+// InitBPFMaps Function
+func (mon *SystemMonitor) initBPFMaps() error {
+	visibilityMap, err := cle.NewMapWithOptions(
+		&cle.MapSpec{
+			Name:       "kubearmor_visibility",
+			Type:       cle.HashOfMaps,
+			KeySize:    8,
+			ValueSize:  4,
+			MaxEntries: 65535,
+			Pinning:    cle.PinByName,
+			InnerMap:   &mon.BpfVisibilityMapSpec,
+		}, cle.MapOptions{
+			PinPath: mon.PinPath,
+		})
+	mon.BpfNsVisibilityMap = visibilityMap
+	mon.UpdateHostVisibility()
+
+	return err
+}
+
+// DestroyBPFMaps Function
+func (mon *SystemMonitor) DestroyBPFMaps() {
+	err := mon.BpfNsVisibilityMap.Unpin()
+	if err != nil {
+		mon.Logger.Warnf("error unpinning bpf map kubearmor_visibility %v", err)
+	}
```

Comment:
return

Reply:
We want to continue to try to close it even if unpin fails